Sources/Extensions/UIKit/Layout.swift

```diff
@@ -44,15 +44,49 @@ public extension UIView {
             return []
         }
 
-        let constrains: [NSLayoutConstraint] = [
+        let constraints: [NSLayoutConstraint] = [
             topAnchor.constraint(equalTo: superview.layoutMarginsGuide.topAnchor),
             leadingAnchor.constraint(equalTo: superview.layoutMarginsGuide.leadingAnchor),
             trailingAnchor.constraint(equalTo: superview.layoutMarginsGuide.trailingAnchor),
             bottomAnchor.constraint(equalTo: superview.layoutMarginsGuide.bottomAnchor),
         ]
 
-        NSLayoutConstraint.activate(constrains)
+        NSLayoutConstraint.activate(constraints)
 
-        return constrains
+        return constraints
+    }
+
+    @discardableResult
+    func centerInSuperview() -> [NSLayoutConstraint] {
```

Comment:
Maybe add `isActive: Bool = true` parameter like `fillInSuperview`?

Reply:
unless we need it, i would prefer not to. Specially since we can add this if needed after without breaking the API (with default arguments)

Reply:
I would say it would have been useful inside `centerAndConstraintInSuperview` 😄